rsearch/config.go

```diff
@@ -0,0 +1,48 @@
+package rsearch
+
+import (
+	"gopkg.in/gcfg.v1"
+)
+
+// Done is an alias for empty struct, used for termination channels
```

Comment:
For that particular use case it's documented practice.

https://blog.golang.org/pipelines

```
func main() {
    in := gen(2, 3)

    // Distribute the sq work across two goroutines that both read from in.
    c1 := sq(in)
    c2 := sq(in)

    // Consume the first value from output.
    done := make(chan struct{}, 2)
    out := merge(done, c1, c2)
    fmt.Println(<-out) // 4 or 9

    // Tell the remaining senders we're leaving.
    done <- struct{}{}
    done <- struct{}{}
}
```